backend/tests/Designer.Tests/Services/OptionsServiceTests.cs

```diff
@@ -18,7 +18,7 @@ public class OptionsServiceTests
     public async Task GetOptionsListIds_ShouldReturnOptionsListIds_WhenOptionsListsExist()
     {
         // Arrange
-        const string repo = "app-with-options"; // Has 2 options lists
+        const string repo = "app-with-options"; // Has 3 options lists
```

Comment:
Should we maybe remove this comment, so that it does not have to be maintained?

Reply:
True! Good point. Will do it 🫡